fullstack/README.md

```diff
@@ -1,18 +1,29 @@
 # Redbubble Fullstack Code Pairing Exercise
 
+Welcome to the fullstack code pairing exercise! You can choose whichever
+language option (Go, Python, or Ruby) that you are comfortable with to pair with
+our engineers during the session.
+
 ## Context
 
-The code under the `price-calculator` folder will be used as a basis for our fullstack pairing exercise.
-In the interview we'll talk through an extension and your thoughts on design and how you might evolve it.
+Each language has its own implementation of a Price Calculator app based on the
+following [functional requirements](http://take-home-test.herokuapp.com/new-product-engineer).
+We'll use it as a basis for our fullstack pairing exercise.
+
+In the interview we'll talk through the structure of the codebase, your thoughts
+on the design, and how you might involve it. Then we will do an extension on the
```

Comment:
Did you mean this?
```suggestion
on the design, and how you might improve it. Then we will do an extension on the
```

Reply:
Good catch! I was thinking about `evolve` but my fingers were involved in the typing 😅